Makefile

```diff
@@ -56,6 +56,9 @@ arch-test:
 .PHONY: all
 all: arch-test finch finch-core finch.yaml networks.yaml config.yaml lima-and-qemu
 
+.PHONY: all-local
+all-local: arch-test finch networks.yaml config.yaml lima-and-qemu finch-core finch.yaml
```

Comment:
Do you want to replace to local-core here?

Reply:
Yeah, typo. Good catch. I changed the names right before I made the PR

Reply:
Fixed in latest rev